.github/PULL_REQUEST_TEMPLATE/BugFix.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug Fix
+about: Add a bug fix this project!
+title: "[Feature] Short Form Title (50 chars or less.)"
+labels: bug
+assignees: 'WithoutPants, bnkai, Leopere'
+
+---
+<!-- Explain what your bugfix seeks to remedy in a short paragraph. -->
+# Scope
+
+<!-- Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick in when this is merged -->
+## This Pull Request Attempts to close the following issues
+
+<!-- Please put an X in the following items to show that you've done your due diligence. -->
+## Due Diligence Checklist
+
+* [ ] I've run Code linters and [gofmt](https://golang.org/cmd/gofmt/) to make sure that my code is readable.
+* [ ] I have read through formerly submitted pull requests and issues to make sure that this is required and so that I can manage to close any Issues needing closed relating to this feature submission.
+* [ ] To the best of my ability, I've followed the [CONTRIBUTING.md](/CONTRIBUTING.md) to make sure that I'm up to date on how to best contribute to Stash projects.
+* [ ] I  commented adequately on my code with the expectation in mind that anyone else should be able to look at this code I've submitted and know exactly what's happening and what the expectations are.
+* [ ] I acknowledge that if applicable to me, submitting and subsequent acceptance of this Pull Request I, the code contributor of this Pull Request, agree and acknowledge my understanding that the new code license has now been updated to [AGPL](/LICENSE.md). I agree that all code before this Pull Request, which I've previously submitted, is now to be re-licensed under the new license AGPL and no longer the former MIT license.
+
+  <!-- Explain why you may not have checked all of the above boxes. -->
+  ### In case I haven't checked all of the boxes above here is my explanation as to why not.
+
+
+<!-- Please check off what platforms that you've tested your build on so that we can potentially skip some steps in accepting your Pull Request -->
+## Platforms that I've tested this build with the following platforms.
+* [ ] Windows 10
+* [ ] OS X Catalina
+* [ ] Linux
+* [ ] Arm
+
```

Comment:
I'd also like to see something about what has been tested by the submitter, and what possible impacts/areas there are that may need retesting by others.

A checkbox is probably unnecessary for platforms, just leave it for the user to fill in. I'd probably make the heading more terse as well - "Platforms tested against" or something.

Reply:
> A checkbox is probably unnecessary for platforms, just leave it for the user to fill in. I'd probably make the heading more terse as well - "Platforms tested against" or something.

This doesn't look like it's been resolved. Having checkboxes in the template also adds a tasks icon to the pull request in the list page, which I think is incorrect.

Reply:
I thought about it for a bit and figured that the checkboxes might imply the platforms that we support vs other platforms that are supported "by the community" perhaps?  Such as these NAS setups and whatnot.

Reply:
It's sufficient to leave that in a comment, then the contributor can copy and paste as needed.

Reply:
I'm not sure but this change requested just isn't sitting right with me. 

There are other fringe benefits as well to this where reviewers can potentially tick off their verified platforms.

Reply:
![image](https://user-images.githubusercontent.com/53250216/74393864-60bbc700-4e5f-11ea-8457-8d22503058b5.png)

This is the issue that I'm referring to. It is a misuse of the tasks functionality. Review platforms are better communicated via explicit comment, rather than anonymous ticking of checkboxes. 

I'd want to hear more benefits of this method of showing tested platforms - to my mind, the benefits given are insufficient.

Reply:
conceded